app/page.tsx

```diff
@@ -0,0 +1,45 @@
+import { stripeClient } from '../utils/client';
+import Products from './Products';
+
+export const metadata = { description: 'Products Page' };
+
+export default async function HomePage() {
+  const tablet = await stripeClient.products.retrieve(process.env.TABLET_ID!);
+  const magazine = await stripeClient.products.retrieve(
+    process.env.MAGAZINE_ID!,
+  );
+
+  const tabletPrice = await stripeClient.prices.retrieve(
+    tablet.default_price as string,
+  );
+  const magazinePrice = await stripeClient.prices.retrieve(
+    magazine.default_price as string,
+  );
+
+  if (!tabletPrice.unit_amount || !magazinePrice.unit_amount) {
+    throw new Error('Invalid Product Price');
```

Comment:
I guess probably we don't want to throw an error in a React component? but rather render an error to the screen?